src/Development/IDE/Core/Rules.hs

```diff
@@ -642,10 +639,10 @@ typeCheckRuleDefinition hsc pm isFoi source = do
 generateCore :: RunSimplifier -> NormalizedFilePath -> Action (IdeResult (SafeHaskellMode, CgGuts, ModDetails))
 generateCore runSimplifier file = do
     deps <- use_ GetDependencies file
-    (tm:tms) <- uses_ TypeCheck (file:transitiveModuleDeps deps)
+    (tm:_) <- uses_ TypeCheck (file:transitiveModuleDeps deps)
```

Comment:
can we drop the `transitiveModuleDeps`?

Reply:
What do you think about deleting this rule?